CMakeLists.txt

```diff
@@ -205,13 +205,13 @@ if (CMAKE_Fortran_COMPILER_ID MATCHES "GNU.*")
     if (CMAKE_Fortran_COMPILER_VERSION VERSION_GREATER_EQUAL 9)
       set (CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fallow-argument-mismatch")
     endif()
-    set (CMAKE_Fortran_FLAGS_RELEASE "-O2")
+    set (CMAKE_Fortran_FLAGS_RELEASE "-O2 -march=native")
```

Comment:
Tested the removal of the `-march=native` flag and [the tests passed](https://github.com/scrasmussen/wrf_hydro_nwm_public/pull/7/commits/e53fca2ef2912ca39b46738b1ebe2491fdc99d27). Seems the differences are just caused by that optimization

Reply:
Great find! We may want to introduce that back eventually (after some benchmarking), as it theoretically can improve performance considerably on some systems.